docs/sources/alerting/add-notification-template.md

```diff
@@ -0,0 +1,23 @@
++++
+title = "Alert notification templating"
+keywords = ["grafana", "documentation", "alerting", "alerts", "notification", "templating"]
+weight = 110
++++
+
+# Alert notification templating
+
+You can provide detailed information to alert notification recipients by injecting alert query data into an alert notification. 
+
+Labels that exist from the evaluation of the alert query can be used in the alert rule name or alert notification message fields. The alert label data is injected into the notification fields when the alert is in the alerting state. When there are multiple values for the same label, the values are comma-separated.
+
+This topic explains how you can use alert query labels in alert notifications.
```

Comment:
Move this to after the intro sentence "You can provide detailed information to alert notification recipients by injecting alert query data into an alert notification."